setup.py

```diff
@@ -9,9 +9,11 @@
 
 gnofract4d_version = '4.2'
 
+_DEBUG = False
+
 if sys.version_info < (3, 6):
     print("Sorry, you need Python 3.6 or higher to run Gnofract 4D.")
-    print("You have version %s. Please upgrade." % sys.version)
+    print("You have version {sys.version}. Please upgrade.")
```

Comment:
I actually deliberately left f-strings out of setup.py, because with this change you won't see the error message about python version if you try to run with py3.5, you'll get a parse error instead. Would prefer to leave this file alone but f-strings anywhere else in the code are fine.

Reply:
I got somewhat carried away :D